### PR TITLE
Feature/edit status

### PR DESCRIPTION
**Pull request by iddanh**

### CODE REVIEW

File: src/services/user.service.js [hunk 1]
```diff
@@ -61,8 +61,57 @@ async function changeNickname({ userId, nickname } = {}) {
   }
 }
 
+function createChangeStatusRequest(status, userIds) {
+  let options = { method: 'POST', json: true, headers: userServiceHeaders };
+  switch (status) {
+    case 'isVerified': {
+      // rp(Object.assign({}, options, {
```
Comment (DanielCiuraru) [inline src/services/user.service.js]: Remove if the comment is not needed

File: src/services/user.service.js [hunk 2]
```diff
@@ -61,8 +61,57 @@ async function changeNickname({ userId, nickname } = {}) {
...
+  try {
+    return await Promise.all(promises);
+
+  } catch (err) {
+    winston.error(`UserService: cannot change status: Error  ${err}`);
+    throw err;
+  }
+}
+
+async function changeManyStatus({ status, userIds } = {}) {
```
Comment (DanielCiuraru) [inline src/services/user.service.js]: name suggestion: updateMultipleUsersToStatus

File: src/services/user.service.js [hunk 3]
```diff
@@ -61,8 +61,57 @@ async function changeNickname({ userId, nickname } = {}) {
...
+    case 'isDelete': {
+      return rp(Object.assign({}, options, {
+        uri: `${pbConfig.get('UserService.url')}delete`,
+        body: { userIds }
+      }));
+    }
+  }
+}
+
+async function changeStatus({ userId, changedStatus } = {}) {
```
Comment (DanielCiuraru) [inline src/services/user.service.js]: name suggestion: updateUsersStatusStates (plural as in more than one status state)

File: src/services/user.service.js (hunk 1 above)
Comment (iddanh) [reply src/services/user.service.js]: This is placeholder

**Pull request merged**